NitroxPatcher/Main.cs

```diff
@@ -28,16 +32,34 @@ public static void Execute()
         {
             Log.Setup(true);
             Optional.ApplyHasValueCondition<Object>(o => (bool)o);
```

Comment:
Potentially explicitly use `UnityEngine.Object` and remove the `using Object = UnityEngine.Object` to avoid possible conflation with C#'s `object`?